subprojects/gradle/impact/build.gradle.kts

```diff
@@ -15,6 +15,7 @@ dependencies {
     implementation(project(":common:math"))
     implementation(project(":gradle:git"))
     implementation(project(":gradle:build-environment"))
+    implementation(project(":gradle:build-metrics"))
```

Comment:
- [ ] extract to a module. Shouldn't consume plugin

Reply:
it should be done in this PR or postponed?

Reply:
Moved to `build-metrics-tracker` module